### ISSUE TITLE
1.20.1-forge-3.1 detects virus

### ISSUE THREAD

**Issue by KralOyuncu55** [ToCraft/Remorphed#22]
### Mod loader

Forge

### Minecraft version

1.20.1-forge-47.2.0

### Remorphed version

1.20.1-forge-3.2

### Woodwalkers version

1.20.1-forge-3.1

### CraftedCore version

1.20.1-forge-3.1

### Modloader version

1.20.1-forge-47.2.0

### The latest.log file

https://gist.github.com/KralOyuncu55/ee00e30ecdeaa3ef5fdd20915f9607a0

### Issue description

1.20.1-forge-3.1 detects virus. latest.log you don't need to look.

### Other

detects virus...

**Comment by Natedogwoooof** [comment]
POV: you use chrome and windows and not Firefox and Linux

I use mint btw been a user for 7 months

**Comment by Natedogwoooof** [comment]
But yeah idk why crash

**Comment by ToCraft** [comment]
> detects virus...

Could you supply a bit more information? For example, where did you download the mod, what message stood there? (a partial screenshot would be nice)

**Comment by ToCraft** [comment]
Alright, I just went through some files and I noticed that the Windows Defender had problems downloading `CraftedCore 3.1-1.20.1-Forge`.
The funny part is, that this file was nearly identical to most other CrafterCore 3.1 files. I actually just re-uploaded it as `Crafted Core 3.1.1-1.20.1-Forge` and it works now. I'm not sure why Windows Defender acted as it did, but I can assure you, that there wasn't a virus in this mod. (All my mods automatically upload via GitHub Actions so every process is publicly visible).
